mmcv/runner/base_runner.py

```diff
@@ -172,6 +173,10 @@ def max_iters(self):
         """int: Maximum training iterations."""
         return self._max_iters
 
+    @abstractmethod
```

Comment:
This method may be just implemented as private method of each subclass.

Reply:
https://github.com/open-mmlab/mmcv/pull/553/commits/97e47f4c3f986d5d1a655e2fc01f58792b6f5d98